src/uu/rm/src/rm.rs

```diff
@@ -326,6 +333,15 @@ pub fn remove(files: &[&OsStr], options: &Options) -> bool {
 fn handle_dir(path: &Path, options: &Options) -> bool {
     let mut had_err = false;
 
+    let path = clean_trailing_slashes(path);
+    if path_is_current_or_parent_directory(path) {
+        show_error!(
+            "refusing to remove '.' or '..' directory: skipping '{:}'",
```

Comment:
```suggestion
            "refusing to remove '.' or '..' directory: skipping '{}'",
```

… or did I misunderstand something about the format-macro?

Reply:
Nah, you didn't misunderstand. I actually was doing debug prints ``:?`` on path during some testing, missed cleaning that part up.